lang/ca/texts.php

```diff
@@ -5301,6 +5301,23 @@
     'latest_requires_php_version' => 'Note: the latest version requires PHP :version',
     'auto_expand_product_table_notes' => 'Automatically expand products table notes',
     'auto_expand_product_table_notes_help' => 'Automatically expands the notes section within the products table to display more lines.',
+    'institution_number' => 'Institution Number',
+    'transit_number' => 'Transit Number',
+    'personal' => 'Personal',
+    'province_code' => 'Province Code',
+    'province_abbreviation' => 'Province Abbreviation',
+    'street_address' => 'Street Address',
+    'full_name' => 'Full Name',
+    'address_information' => 'Address Information',
+    'enter_the_information_for_the_bank_account' => 'Enter the Information for the Bank Account',
+    'address_line_1' => 'Address Line 1',
+    'address_line_2' => 'Address Line 2',
+    'account_holder_information' => 'Account Holder Information',
+    'enter_information_for_the_account_holder' => 'Enter Information for the Account Holder',
+    'home_phone' => 'Home Phone',
+    'customer_type' => 'Customer Type',
+    'process_date' => 'Process Date',
+    'other_phone' => 'Other Phone'
```

Comment:
only the en/texts.php file is needed, also there are duplicate here, we need to reuse any existing translations and only add in ones that are not already present in the translation file.

Reply:
Not sure what these duplicates are. Can you point them out to me. Note that I notice this pull request isn't using the most recent code updates. Should I make a new one?

Reply:
![image](https://github.com/user-attachments/assets/f7ff7c85-4dda-4095-9ecd-0dec913c285c)

We always try to reuse / keep consistent the translations. We already have ~5k translation so we try and be as prudent as possible

Reply:
Hmmm. I see the test was based on Rotessa's fields and the fact that it uses "Canadian English Language" This may differ in context from US/ other english language. If you are suggesting I use the "other language" as a base instead I would but would this not affect the context and language translations?

Reply:
We only support English and English - United Kingdom. 

The translations you have added here have been put in the CA language which is Catalan. Also for any language other than en, we manage the translations in Transifex here https://transifex.com

If English - Canadian as a language wants to be added, this would need to be added into Transifex, and then the translations would need to be perfomed on all strings until they were at least 70% completed prior to importing that in as a new language.

Reply:
Ok I see. So to be clear...

-  I need to update either **en_US** or **en_GB** language file.
-  I should also try to use what's already there? e.g **'texts.address1'** as oppose to **'texts.address_line_1'**
-  I should try to use a _close match_ as oppose to adding a **custom match** where possible? e.g. "postal_code' => 'Postcode' instead of me adding **'ca_postal_code' => 'Postal Code'** ?

Would need to report back so just need to be sure.

Reply:
@karneaud only the en/texts.php file is ever updates with new translations, thats correct. The remaining points are also correct. If you find a match in the texts.php file, please use it, if there is no close match, then the new translation can be added to en/texts.php